lib/utils/UIConstants.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:ui';
+
+abstract class UIConstants {
+  ///Colors used
+  static const Color blueGrey900 = Color(0xFF263238);
+  static const Color black54 = Color(0x8A000000);
+  static const Color transparent = Color(0x00000000);
+  static const Color white = Color(0xFFFFFFFF);
+  static const Color white60 = Color(0x99FFFFFF);
+  static const Color blueGrey800 = Color(0xFF37474F);
+
+  ///Details page Background color
+  static const Color detailsBackgroundColor = blueGrey900;
+
+  ///Details page Backdrop
+  static const double backdropAspectRatio = 16/9;
+  static const List<double> backdropGradientStops = [
+    0,
+    0.1,
+  ];
+  static const List<Color> backdropGradientColors = [
+    black54,
+    transparent,
+  ];
+
+  ///Details page Title section
+  static const Color titleSectionTextColor = white;
+  static const double titleTextAreaPadding = 8;
+
+  ///Details page Poster section
+  static const double posterSectionSize = 280;
+  static const double posterSectionIconSize = 120;
+
+
+  ///Details page Overview section
+  static const Color overviewSectionTextColor = white60;
+  static const Color overviewTitleBackgroundColor = blueGrey800;
```

Comment:
Here you are creating a const value that has assign a const value too, you can use directly white60 and blueGrey800

Reply:
I decided to remove the blueGray800 and white60 constants and instead assign those Colors directly to the overview.... Color constants, I find it clearer when the name describes where the color is used. Is it wrong to do this?